mappings/net/minecraft/screen/SmithingScreenHandler.mapping

```diff
@@ -1,8 +1,18 @@
 CLASS net/minecraft/class_4862 net/minecraft/screen/SmithingScreenHandler
+	FIELD field_25385 world Lnet/minecraft/class_1937;
+	FIELD field_25386 recipe Lnet/minecraft/class_5357;
```

Comment:
`selectedRecipe`?

Reply:
I'll make it `currentRecipe` to be more clear. You don't 'select' Smithing Recipes.